docs/eventing/samples/apache-rocketmq-source/README.md

```diff
@@ -0,0 +1,199 @@
+* [Knative Eventing v0.9+](https://knative.dev/docs/install/any-kubernetes-cluster/)
+* [Knative Serving v0.9+](https://knative.dev/docs/install/any-kubernetes-cluster/)
+* [Apache RocketMQ cluster](http://rocketmq.apache.org/docs/quick-start/)
+
+
+## Creating a `RocketMQSource` source CRD
+
+1. Install the `rocketmqsource ` sub-component to your Knative cluster:
+
+   ```
+   kubectl apply -f https://raw.githubusercontent.com/apache/rocketmq-externals/master/rocketmq-knative/source/config/200-serviceaccount.yaml \
+                  -f https://raw.githubusercontent.com/apache/rocketmq-externals/master/rocketmq-knative/source/config/201-clusterrole.yaml \
+                  -f https://raw.githubusercontent.com/apache/rocketmq-externals/master/rocketmq-knative/source/config/202-clusterrolebinding.yaml \
+                  -f https://raw.githubusercontent.com/apache/rocketmq-externals/master/rocketmq-knative/source/config/300-rocketmqsource.yaml \
+                  -f https://raw.githubusercontent.com/apache/rocketmq-externals/master/rocketmq-knative/source/config/400-controller-service.yaml \
+                  -f https://raw.githubusercontent.com/apache/rocketmq-externals/master/rocketmq-knative/source/config/500-controller.yaml \
+                  -f https://raw.githubusercontent.com/apache/rocketmq-externals/master/rocketmq-knative/source/config/600-istioegress.yaml
+
+   ```
+2. Check that the `rocketmqsource-controller-manager-0` pod is running.
+
+   ```
+   $ kubectl -n knative-sources  get pods
+      NAME                                 READY     STATUS    RESTARTS   AGE
+      rocketmqsource-controller-manager-0   1/1       Running   0          10s
+   ```
+3. Check the `rocketmqsource-controller-manager-0` pod logs.
+
+
+   ```
+   $ kubectl -n knative-sources logs rocketmqsource-controller-manager-0
+      2020/03/13 04:37:57 Registering Components.
+      2020/03/13 04:37:57 Setting up Controller.
+      2020/03/13 04:37:57 Adding the RocketMQ Source controller.
+      2020/03/13 04:37:57 Starting rocketmqsource controller.
+   ```
+## Create the event display service
+
+1. Deploy the event display service
+
+   ```
+   $ kubectl apply -f https://raw.githubusercontent.com/apache/rocketmq-externals/master/rocketmq-knative/source/sample/event-display.yaml
+      service.serving.knative.dev/event-display created
+   ```
+1. Ensure that the Service pod is running. The pod name will be prefixed with `event-display`.
+   ```
+   $ kubectl get pods
+      NAME                                            READY     STATUS    RESTARTS   AGE
+      event-display-6g8pr-deployment-56897dcdfb-vcm82   2/2     Running   0          28s
+
+   ```
+
+## Create Apache RocketMQ Event Source
+
+1. Modify [`apacharocketmqsource.yaml`](https://github.com/apache/rocketmq-externals/blob/master/rocketmq-knative/source/sample/apacherocketmqsource.yaml)accordingly with namesrvAddr,topics,etc.:
+
+   ```yaml
+  apiVersion: sources.eventing.knative.dev/v1alpha1
```

Comment:
we've been recently moving to sources.knative.dev/v1alpha1, might be worth considering doing so as well
You can check examples in eventing-contrib

Reply:
As you said, I have seen eventing-contrib. Most of sources use sources.knative.dev/v1alpha1, such as kafkasource, camal source, etc.